client/src/luigi-client.js

```diff
@@ -134,20 +113,20 @@ function luigiClientInit() {
     },
     '*'
   );
-}
+};
 
 luigiClientInit();
 
-/** @namespace */
+/** @private */
 const LuigiClient = {
   /**
-   * Use the functions and parameters to define the lifecycle of listeners, navigation nodes, and Event data.
-   * @name lifecycle
+   * Use the functions and parameters to define the Lifecycle of listeners, navigation nodes, and Event data.
```

Comment:
As discussed, let´s move the lifecycle methods to its own class ;)

Reply:
done